niyobobo/PoliticoReact
# #167286097 User should be able to login

### What does this PR do?
User should be able to login
### Description of Task to be completed?
- Configure redux
- Create a user login page
### How should this be manually tested?
- Clone the repository locally with `git clone git@github.com:niyobobo/PoliticoReact.git` command.
- Check out to this branch by running `git checkout ft-user-login-167286097`.
- Install dependencies by running `npm install`.
- To start the server run `npm start`, you will be directed to this URL`http://localhost:8000/` in your browser.
- Provide email and password to login
### Any background context you want to provide?
To be able to log in you must have an account on this platform.
### What are the relevant pivotal tracker stories?
[#167286097](https://www.pivotaltracker.com/story/show/167286097)
### Screenshot:
![image](https://user-images.githubusercontent.com/29593858/61235624-3aea0a80-a736-11e9-89f8-f608196bc96e.png)
